packages/jss/src/utils/escape.js

```diff
@@ -1,10 +1,9 @@
-const env = process.env.NODE_ENV
 const escapeRegex = /([[\].#*$><+~=|^:(),"'`\s])/g
 
 export default str => {
   // We don't need to escape it in production, because we are not using user's
   // input for selectors, we are generating a valid selector.
-  if (env === 'production') return str
+  if (process.env.NODE_ENV === 'production') return str
```

Comment:
We might need to investigate if this has performance problems on the server because accessing environment variables is expensive.

Reply:
its not expensive, it will be replaced by a string and in prod after uglify removed completely

Reply:
Yeah, I don't think its expensive, its a regular property.